vite.base.config.ts

```diff
@@ -1,19 +1,13 @@
 import { builtinModules } from "node:module";
 import type { AddressInfo } from "node:net";
 import type { ConfigEnv, Plugin, UserConfig } from "vite";
-import pkg from "./package.json";
 
 export const builtins = [
   "electron",
   ...builtinModules.map((m) => [m, `node:${m}`]).flat(),
 ];
 
-export const external = [
-  ...builtins,
-  ...Object.keys(
-    "dependencies" in pkg ? (pkg.dependencies as Record<string, unknown>) : {},
-  ),
-];
+export const external = [...builtins];
```

Comment:
Why was this removed?

Reply:
This was removed because it was preventing react from being packaged in the final distro causing a "cannot find module react" error. I am using react in order to be able to utilize useEffect so that the ui can be responsive to the "useCustomUrl" switch. 

I don't think this line was doing anything meaningful anyway, as removing it does not change the final build size appreciably from what I saw.

Reply:
This removal was to address my comment above:

> Looks like it's struggling to find react. I'm having trouble fixing it beucase I'm not quite as familiar with typescript but will keep trying